.github/workflows/deploy.yml

```diff
@@ -41,12 +41,12 @@ jobs:
             token: ${{ secrets.PAT_TO_RUN_AS_ERIC }}
               
         - name: Setup PostgreSQL with PostgreSQL extensions and unprivileged user
-          uses: Daniel-Marynicz/postgresql-action@1.0.0
+          uses: ikalnytskyi/action-setup-postgres@v7
```

Comment:
Why does the deploy stage need the DB?

Reply:
We needed it for running the precompile as that has been required by Rails in the past. It might not be any more.

Reply:
Actually, I'll just remove it and see what happens :smile:

Reply:
@caseyhelbling I tried to remove the postgres action and `bin/setup` but it turns out when we run `assets:precompile` it needs to connect to the DB. So we do need to have a postgres db running to do that.